docs/tutorial.md

```diff
@@ -709,3 +727,21 @@ namespace Pets {
 }
 
 ```
+
+### CADL Config
+
+Cadl has a configuration file that right now is only used to configure the default emitter to use.
+The config file needs to be a sibling of the `package.json`. Cadl will look for the following files in that order and pick the 1st one found:
+
+- `.cadlrc.yaml`: Config as yaml
+- `.cadlrc.yml`: Config as yaml
+- `.cadlrc.json`: Config as json
+- `package.json`: Config as json under a `cadl` property.
```

Comment:
Those the existing cadl config naming from when the cadl config was introduced. Thinking it might be better to have actually a single way of loading the config and a name that feels more like a project config. Maybe:
- `cadlconfig.yaml` 
- `cadl-project.yaml`

Reply:
No strong opinion here. It's maybe a little weird that package.json is not hidden but this is. I'm OK with either of those alternatives.

Reply:
Updated to `cadl-project.yaml` and added `extends` functionallity which I think is going to be needed for our rpaas samples where we want to emit the rpaas controller as well.